src/Runner/DefaultTestResultCache.php

```diff
@@ -178,17 +180,24 @@ public function serialize(): string
         ]);
     }
 
+    /**
+     * @param string $serialized
```

Comment:
Is there a reason to annotate the type in a docblock instead of denoting it in syntax?

Reply:
Yes, because the declaration needs to be the same as the declaration of PHP's own `unserialize` (which does not have a type declaration). If I add a type declaration, PhpStorm complains, and running the tests is not possible anymore at all.

Reply:
I see, or rather I did not see that `\Serializable` is implemented here.